packages/app-page-builder/src/admin/views/PageBlocks/BlocksByCategoriesDataList.tsx

```diff
@@ -379,7 +369,7 @@ const BlocksByCategoriesDataList = ({ canCreate }: PageBuilderBlocksByCategories
                 {({ data }: { data: PbBlockCategory[] }) => (
                     <ScrollList data-testid="default-data-list">
                         {data.map(item => {
-                            const numberOfBlocks = pageBlocksData.filter(
+                            const numberOfBlocks = blocksList.filter(
```

Comment:
Do we really need to do this filter here? Can't we just use the data returned from `useCategoriesListData`?

So, `blocksList.length` ?

Sorry if I'm missing something.

Reply:
@adrians5j 
`useCategoriesListData` returns us full list of filtered blocks. This filter is used to get numberOfBlocks per category